extensions/wikia/CuratedContent/ImageFinderController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+class ImageFinderController extends WikiaApiController {
+	public function getList() {
+		$query = $this->getRequiredParam( 'query' );
+		$limit = $this->getVal( 'limit', 24 );
+
+		$request = new WebRequest();
+		$request->setVal( 'query', $query );
+		$request->setVal( 'limit', $limit );
+		$request->setVal( 'format', 'array' );
+		$request->setVal( 'ns', NS_FILE );
+
+		$suggestions = LinkSuggest::getLinkSuggest( $request );
+		if ( !is_array( $suggestions ) ) {
+			$suggestions = [];
+		}
+
+		$results = [];
+		$filePageIds = array_map( 'intval', array_keys( $suggestions ) );
+		$titles = Title::newFromIDs( $filePageIds );
+
+		foreach ( $titles as $title ) {
+			$file = wfFindFile( $title );
+			if ( $file &&
+				 $file instanceof LocalFile &&
```

Comment:
same note as UCP repo

Reply:
done in https://github.com/Wikia/app/pull/17545/commits/6e3b03be2ed543b7cce979fa1da0a698987c80ed